src/h5web/providers/jupyter/models.ts

```diff
@@ -0,0 +1,43 @@
+import type { HDF5Value } from '../hdf5-models';
+import { EntityKind } from '../models';
+
+export interface JupyterMetaResponse {
+  name: string;
+  type: EntityKind.Group | EntityKind.Dataset;
+}
+
+export interface JupyterMetaDatasetResponse extends JupyterMetaResponse {
+  dtype: string;
+  ndim: number;
+  shape: number[];
+  size: number;
+  type: EntityKind.Dataset;
+}
+
+export interface JupyterMetaGroupResponse extends JupyterMetaResponse {
+  type: EntityKind.Group;
+}
+
+export interface JupyterContentDatasetResponse {
+  content: JupyterMetaDatasetResponse;
+  name: string;
+  type: EntityKind.Dataset;
+  uri: string;
+}
+
+export interface JupyterContent {
+  type: EntityKind;
+  name: string;
+  uri: string;
+  content: null;
+}
```

Comment:
Could you remind me what the term "content" refers to? I'm confused.

Reply:
I think it is not clear yet :smile:

Reply:
The *content* endpoint is to recover `name`, `type` and `uri` of children entities of a group. 

For a dataset, it returns the `name`, `type` and `uri` **of the dataset** as well as its metadata (that you could get through the *meta* endpoint) in a `content` field.

Reply:
Alright, shame that `fetchMetadata` doesn't already return the URIs of the children. It's fine for now of course 👍